gtests/net/mptcp/fastopen/client-TCP_FASTOPEN_CONNECT.pkt

```diff
@@ -16,7 +16,10 @@
 +0.0      >   .  1:1(0)  ack 1             <nop, nop,         TS val 101 ecr 700, dss dack4=1 dsn8=1 ssn=0 dll=1 nocs fin, nop, nop>
 +0.0      <   .  1:1(0)  ack 1  win 450    <nop, nop,         TS val 700 ecr 100, dss dack4=2 dsn4=1 ssn=0 dll=1 nocs fin, nop, nop>
 +0.0      >   .  1:1(0)  ack 1             <nop, nop,         TS val 101 ecr 700, dss dack4=2 nocs>
-+0.0      >  R.  1:1(0)  ack 1             <nop, nop,         TS val 101 ecr 700, mp_fastclose, mp_reset 0>
++0.0      >  F.  1:1(0)  ack 1             <nop, nop,         TS val 101 ecr 700, dss dack4=2 nocs>
++0.0      <   .  1:1(0)  ack 2  win 450    <nop, nop,         TS val 700 ecr 100, dss dack4=2 nocs>
++0.0      <  F.  1:1(0)  ack 2  win 450    <nop, nop,         TS val 700 ecr 100, dss dack4=2 nocs>
++0.0      >   .  2:2(0)  ack 2             <nop, nop,         TS val 101 ecr 700>
```

Comment:
we need to do the same modification in `client-MSG_FASTOPEN.pkt` but I can do that if you prefer

Reply:
I just updated your branch, this is fixed now